packages/recommend-core/src/utils/mapToRecommendations.ts

```diff
@@ -1,22 +1,55 @@
-import { sortBy } from './sortBy';
+import { ProductRecord } from '../types';
+import { AverageIndexes } from '../types/AverageIndexes';
+import { IndexTracker } from '../types/IndexTracker';
+
+import { getAverageIndexes } from './computeAverageIndexes';
 
 type MapToRecommendations<TObject> = {
-  hits: Array<TObject & { _score?: number }>;
+  hits: Array<Array<ProductRecord<TObject>>>;
   maxRecommendations?: number;
+  nrOfObjs: number;
 };
 
 export function mapToRecommendations<TObject>({
   hits,
   maxRecommendations,
+  nrOfObjs,
 }: MapToRecommendations<TObject>) {
-  // Since recommendations from multiple indices are returned, we
-  // need to sort them descending based on their score.
-  return sortBy<TObject & { _score?: number }>((a, b) => {
-    const scoreA = a._score || 0;
-    const scoreB = b._score || 0;
+  const indexTracker: IndexTracker = {};
+
+  // eslint-disable-next-line array-callback-return
+  hits.map((arr) => {
```

Comment:
If you don't return anything you can use a forEach :). Like that you might remove `array-callback-return`

Reply:
ah good point, thank you